### PR TITLE
moved data into seperate files

### LINKED ISSUE #3: Move data into separate files
Instead of using objects for projects & students we can store them in a csv file and import them wherever required as json object using [csvtojson](https://www.npmjs.com/package/csvtojson)

### PR DESCRIPTION
**Pull request by sohan2410**
Instead of keeping projects, students, studentScoreCard in JSON, I've exported them as objects in js which will reduce importing of 'fs'
Fixes #3

### CODE REVIEW

File: public/projects.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export const projects = [
```
Comment (hs2361) [inline public/projects.js:1]:
```suggestion
export default [
```
Use default export

File: public/studentScoreCard.js [hunk 2]
```diff
@@ -0,0 +1,823 @@
+export const students = {
```
Comment (hs2361) [inline public/studentScoreCard.js:1]:
```suggestion
export default {
```
Use default export

File: public/students.js [hunk 3]
```diff
@@ -0,0 +1,517 @@
+export const students = [
```
Comment (hs2361) [inline public/students.js:1]:
```suggestion
export default [
```
Use default export

File: src/server.js [hunk 4]
```diff
@@ -9,15 +9,15 @@ app.use(cors());
 const __dirname = path.resolve();
 
 app.get("/", (req, res) => {
-  fs.readFile(path.join(__dirname, "/src/score.json"), (err, data) => {
+  fs.readFile(path.join(__dirname, "/public/score.json"), (err, data) => {
```
Comment (hs2361) [inline src/server.js:12]:
```suggestion
  fs.readFile(path.join(__dirname, "public/score.json"), (err, data) => {
```
Remove leading / from path as we will call `npm start` from the root of the project

File: src/updaterDB.js [hunk 5]
```diff
@@ -1,861 +1,12 @@
...
-    name: "Devesu Sundram Dev",
-    rollNumber: "2020IMT-027",
-    score: 0,
-    easy: 0,
-    medium: 0,
-    hard: 0,
-  },
-};
+import projects from "../public/projects";
+import students from "../public/studentScoreCard";
```
Comment (hs2361) [inline src/updaterDB.js:5]:
```suggestion
import projects from "../public/projects.js";
import students from "../public/studentScoreCard.js";
```
Add file extension, as without it a module not found error is being thrown

Comment (sohan2410) [reply src/updaterDB.js:5]: Since both the file extensions are same, we need not specify `.js`

Comment (hs2361) [reply src/updaterDB.js]: @sohan2410 You do need to add the .js file extension. We have set our type to be a module, which means the extension is required. Take a look at [this StackOverflow answer](https://stackoverflow.com/a/67105989).

Comment (sohan2410) [reply src/updaterDB.js]: Okay, I've made changes.

File: src/updaterDB.js [hunk 6]
```diff
@@ -905,7 +56,7 @@ const main = async () => {
   for (let i = 0; i < projects.length; i++) {
     await getProjects(projects[i]);
   }
-  fs.writeFile("./src/score.json", JSON.stringify(score, null, 2), (err) => {
+  fs.writeFile("./public/score.json", JSON.stringify(score, null, 2), err => {
```
Comment (hs2361) [inline src/updaterDB.js:59]: Change this path to `../public/score.json`

**Review by hs2361 — CHANGES_REQUESTED** [review]

File: src/updaterDB.js (hunk 6 above)
Comment (sohan2410) [reply src/updaterDB.js]: done

**Pull request merged**